plugin.audio.radiofrance/addon.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.radiofrance" name="RadioFrance — Lives and Podcasts" version="1.0.0" provider-name="bateast">
+    <requires>
+        <import addon="script.module.requests" version="2.25.1" />
+        <import addon="xbmc.python" version="3.0.0" />
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="default.py">
+        <provides>audio</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en">https://www.radiofrance.fr</summary>
+        <license>GNU General Public License, v3</license>
+        <description lang="en">Get media, livestations and podcasts from RadioFrance (https://www.radiofrance.fr/podcasts)</description>
+        <website>https://github.com/bateast/plugin.audio.podcastrf</website>
```

Comment:
This URL redirects: https://github.com/bateast/plugin.audio.podcastrf redirects to https://github.com/bateast/plugin.audio.radiofrance. Please update it with the correct one.

Reply:
ok